tez-runtime-internals/src/main/java/org/apache/tez/runtime/task/TaskRunner2Callable.java

```diff
@@ -116,6 +120,11 @@ public TaskRunner2CallableResult run() throws Exception {
       // For a successful task, however, this should be almost no delay since close has already happened.
       maybeFixInterruptStatus();
       LOG.info("Cleaning up task {}, stopRequested={}", task.getTaskAttemptID(), stopRequested.get());
+      String ioStats = IOStatisticsLogging.ioStatisticsToPrettyString(
```

Comment:
looks good. you know you can take a snapshot of it and serialize it as java serializable or json; been some discussion about making a Writable too.